2017/07/31 04:12:03 [error] 5#5: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: admin.statsdigital.dev, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "admin.statsdigital.dev:8000"
2017/07/31 04:12:05 [error] 5#5: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: admin.statsdigital.dev, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "admin.statsdigital.dev:8000"
2017/07/31 04:12:06 [error] 5#5: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: admin.statsdigital.dev, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "admin.statsdigital.dev:8000"
2017/08/21 17:38:48 [error] 5#5: *53 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\web\ForbiddenHttpException: Login Required in /web/stats-digital/vendor/yiisoft/yii2/web/User.php:427
Stack trace:
#0 /web/stats-digital/vendor/yiisoft/yii2/filters/AccessControl.php(152): yii\web\User->loginRequired()
#1 /web/stats-digital/vendor/yiisoft/yii2/filters/AccessControl.php(137): yii\filters\AccessControl->denyAccess(Object(yii\web\User))
#2 /web/stats-digital/vendor/yiisoft/yii2/base/ActionFilter.php(75): yii\filters\AccessControl" while reading upstream, client: 172.18.0.1, server: admin.statsdigital.local, request: "GET /site/login HTTP/1.1", upstream: "fastcgi://172.18.0.3:9000", host: "admin.statsdigital.local"
2017/08/21 17:38:48 [error] 5#5: *53 FastCGI sent in stderr: "->beforeAction(Object(yii\web\ErrorAction))
#3 [internal function]: yii\base\ActionFilter->beforeFilter(Object(yii\base\ActionEvent))
#4 /web/stats-digital/vendor/yiisoft/yii2/base/Component.php(545): call_user_func(Array, Object(yii\base\ActionEvent))
#5 /web/stats-digital/vendor/yiisoft/yii2/base/Controller.php(272): yii\base\Component->trigger('beforeAction', Object(yii\base\ActionEvent))
#6 /web/stats-digital/vendor/yiisoft/yii2/web/Controller.php(164): yii\base\Controller->beforeAction(Object(yii\w" while reading upstream, client: 172.18.0.1, server: admin.statsdigital.local, request: "GET /site/login HTTP/1.1", upstream: "fastcgi://172.18.0.3:9000", host: "admin.statsdigital.local"
2017/08/21 17:38:52 [error] 5#5: *53 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\web\ForbiddenHttpException: Login Required in /web/stats-digital/vendor/yiisoft/yii2/web/User.php:427
Stack trace:
#0 /web/stats-digital/vendor/yiisoft/yii2/filters/AccessControl.php(152): yii\web\User->loginRequired()
#1 /web/stats-digital/vendor/yiisoft/yii2/filters/AccessControl.php(137): yii\filters\AccessControl->denyAccess(Object(yii\web\User))
#2 /web/stats-digital/vendor/yiisoft/yii2/base/ActionFilter.php(75): yii\filters\AccessControl" while reading upstream, client: 172.18.0.1, server: admin.statsdigital.local, request: "GET /site/login HTTP/1.1", upstream: "fastcgi://172.18.0.3:9000", host: "admin.statsdigital.local"
2017/08/21 17:38:52 [error] 5#5: *53 FastCGI sent in stderr: "->beforeAction(Object(yii\web\ErrorAction))
#3 [internal function]: yii\base\ActionFilter->beforeFilter(Object(yii\base\ActionEvent))
#4 /web/stats-digital/vendor/yiisoft/yii2/base/Component.php(545): call_user_func(Array, Object(yii\base\ActionEvent))
#5 /web/stats-digital/vendor/yiisoft/yii2/base/Controller.php(272): yii\base\Component->trigger('beforeAction', Object(yii\base\ActionEvent))
#6 /web/stats-digital/vendor/yiisoft/yii2/web/Controller.php(164): yii\base\Controller->beforeAction(Object(yii\w" while reading upstream, client: 172.18.0.1, server: admin.statsdigital.local, request: "GET /site/login HTTP/1.1", upstream: "fastcgi://172.18.0.3:9000", host: "admin.statsdigital.local"
2017/08/21 17:39:43 [error] 5#5: *53 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\web\ForbiddenHttpException: Login Required in /web/stats-digital/vendor/yiisoft/yii2/web/User.php:427
Stack trace:
#0 /web/stats-digital/vendor/yiisoft/yii2/filters/AccessControl.php(152): yii\web\User->loginRequired()
#1 /web/stats-digital/vendor/yiisoft/yii2/filters/AccessControl.php(137): yii\filters\AccessControl->denyAccess(Object(yii\web\User))
#2 /web/stats-digital/vendor/yiisoft/yii2/base/ActionFilter.php(75): yii\filters\AccessControl" while reading upstream, client: 172.18.0.1, server: admin.statsdigital.local, request: "GET /site/login HTTP/1.1", upstream: "fastcgi://172.18.0.3:9000", host: "admin.statsdigital.local"
2017/08/21 17:39:43 [error] 5#5: *53 FastCGI sent in stderr: "->beforeAction(Object(yii\web\ErrorAction))
#3 [internal function]: yii\base\ActionFilter->beforeFilter(Object(yii\base\ActionEvent))
#4 /web/stats-digital/vendor/yiisoft/yii2/base/Component.php(545): call_user_func(Array, Object(yii\base\ActionEvent))
#5 /web/stats-digital/vendor/yiisoft/yii2/base/Controller.php(272): yii\base\Component->trigger('beforeAction', Object(yii\base\ActionEvent))
#6 /web/stats-digital/vendor/yiisoft/yii2/web/Controller.php(164): yii\base\Controller->beforeAction(Object(yii\w" while reading upstream, client: 172.18.0.1, server: admin.statsdigital.local, request: "GET /site/login HTTP/1.1", upstream: "fastcgi://172.18.0.3:9000", host: "admin.statsdigital.local"
2017/08/21 17:40:00 [error] 5#5: *53 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\web\ForbiddenHttpException: Login Required in /web/stats-digital/vendor/yiisoft/yii2/web/User.php:427
Stack trace:
#0 /web/stats-digital/vendor/yiisoft/yii2/filters/AccessControl.php(152): yii\web\User->loginRequired()
#1 /web/stats-digital/vendor/yiisoft/yii2/filters/AccessControl.php(137): yii\filters\AccessControl->denyAccess(Object(yii\web\User))
#2 /web/stats-digital/vendor/yiisoft/yii2/base/ActionFilter.php(75): yii\filters\AccessControl" while reading upstream, client: 172.18.0.1, server: admin.statsdigital.local, request: "GET /site/login HTTP/1.1", upstream: "fastcgi://172.18.0.3:9000", host: "admin.statsdigital.local"
2017/08/21 17:40:00 [error] 5#5: *53 FastCGI sent in stderr: "->beforeAction(Object(yii\web\ErrorAction))
#3 [internal function]: yii\base\ActionFilter->beforeFilter(Object(yii\base\ActionEvent))
#4 /web/stats-digital/vendor/yiisoft/yii2/base/Component.php(545): call_user_func(Array, Object(yii\base\ActionEvent))
#5 /web/stats-digital/vendor/yiisoft/yii2/base/Controller.php(272): yii\base\Component->trigger('beforeAction', Object(yii\base\ActionEvent))
#6 /web/stats-digital/vendor/yiisoft/yii2/web/Controller.php(164): yii\base\Controller->beforeAction(Object(yii\w" while reading upstream, client: 172.18.0.1, server: admin.statsdigital.local, request: "GET /site/login HTTP/1.1", upstream: "fastcgi://172.18.0.3:9000", host: "admin.statsdigital.local"
